src/Pact/Analyze/Translate.hs

```diff
@@ -1614,8 +1614,17 @@ translateNode astNode = withAstContext astNode $ case astNode of
         pure $ Some SBool $ Lit' True
       _ -> failing $ "Pattern match failure"
 
+  AST_NFun node "diff-time" [a,b] -> do
+    t0 <- translateNode a
+    t1 <- translateNode b
+    case (t0, t1) of
+      (Some STime _, Some STime _) -> do
+        addWarning node $ UnsupportedNonFatal "diff-time: substituting true expression"
```

Comment:
```suggestion
        addWarning node $ UnsupportedNonFatal "diff-time: substituting 0"
```

Reply:
I went with 

```haskell
 addWarning node $ UnsupportedNonFatal "diff-time: substituting 0.0"
```

So you get the correct type in the error. Otherwise, seems good to go!